data/fields/entrance.json

```diff
@@ -1,5 +1,19 @@
 {
     "key": "entrance",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+    "strings": {
+        "options": {
+          "yes": "Unspecified",
```

Comment:
Some possibly relevant naming considerations for entrance types in streetcomplete/StreetComplete#3949.